lib/features/onboarding/ui/pages/financial_info_page.dart

```diff
@@ -78,10 +78,13 @@ class _FinancialInfoPageState extends State<FinancialInfoPage> {
         ),
         BottomActionBar(
           label: OnboardingConstants.confirmButtonText,
-          onPressed: () => widget.onTap.call(
-            selectedIncome,
-            selectedEmploymentStatus,
-          ),
+          onPressed:
+              (selectedIncome != null && selectedEmploymentStatus != null)
```

Comment:
Create an method and call at ontap & also use same naming convention everywhere i.e, `onTap` instead of onPressed or anything else 

```suggestion
              _onTapConfirm
```

Reply:
Done